package-testing/testing-api/src/routes.ts

```diff
@@ -51,7 +51,15 @@ routes.get('/api/flag-config/v1/bandits', (req, res) => {
   }
 
   return res.status(404).json({ message: 'No data for specified SDK' });
-});
+};
+
+// Serve with and without the /api prefix to accomodate SDKs in transition.
+routes.get('/api/flag-config/v1/config', serveUFC);
+routes.get('/api/flag-config/v1/bandits', serveBanditParams);
+
+// For SDKs which in transition; they're passed a baseUrl including `/api` and are using endpoint constants which start with `api/`
```

Comment:
nice thinking.

Also, glad we are getting consistent here--thanks for pushing us in that direction!